src/components/SubmitButton/SubmitButton.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components/macro'
+export default function SubmitButton() {
+  return <Button onclick={handleSubmitFilter}> Submit </Button>
+
+  function handleSubmitFilter() {}
```

Comment:
It looks like you are not using this component. If you don't need it, you should get rid of this and the test and storie files which are not working because you didn't name them properly ;)